### DISCUSSION TITLE
How do you set conditions for real-time data monitoring?

### DISCUSSION THREAD

**Discussion by jhh78** [pocketbase/pocketbase#4330] [Q&A] [answered]
greetings.
Thank you for providing this great feature called real-type monitoring.

I want to monitor only common keys in a certain collection. Please tell me how to do this.

Please tell me how to write to monitor the following conditions.
post collection : id, title, contents
reply collection : id, postTableId, contents

post.id = reply.postTableId

**Comment by ganigeorgiev** [comment, marked as answer]
It is not clear to me what do you mean by _"monitor only common keys in a certain collection"_.

If you want to listen to the changes of all replies that are assigned to a specific `postTableId` then you can try specifying the `filter` parameter:

```js
const postId = "..."

await pb.collection("reply").subscribe("*", (e) => {
    ...
}, {
    filter: `postTableId="${postId}"`
})

```
